ios/Classes/PurchasesFlutterPlugin.m

```diff
@@ -201,6 +203,9 @@ - (void)handleMethodCall:(FlutterMethodCall *)call
         NSNumber *callbackID = arguments[@"callbackID"];
         [self startPromotedProductPurchase:callbackID
                                     result:result];
+    } else if ([@"showInAppMessages" isEqualToString:call.method]) {
+        NSSet<NSNumber*>* types = arguments[@"types"];
```

Comment:
Oh interesting, so this converts directly to an `NSSet`? I thought it would be an NSArray and we can convert it internally to an NSSet

Reply:
Actually, I was overly optimistic about the casting. It doesn't do proper automatic conversion :D good catch